src/index.tsx

```diff
@@ -485,9 +490,7 @@ class Formsy extends React.Component<FormsyProps, FormsyState> {
     // If there are no inputs, set state where form is ready to trigger
     // change event. New inputs might be added later
     if (!this.inputs.length) {
-      this.setState({
-        canChange: true,
-      });
+      onValidationComplete();
```

Comment:
Oh, and i also fixed a bug, where if you remove all the inputs but the last one was invalid, the form was still invalid. The callback onValidationComplete was never called.